src/controller/java/src/chip/devicecontroller/ICDDeviceInfo.java

```diff
@@ -17,16 +17,129 @@
  */
 package chip.devicecontroller;
 
+import java.util.HashSet;
+import java.util.Set;
+
 /** Class for holding ICD Device information. */
 public class ICDDeviceInfo {
-  private byte[] symmetricKey;
+  /** List of methods to trigger the ICD to switch to Active mode. */
+  public enum UserActiveModeTriggerBitmap {
+    PowerCycle(0),
```

Comment:
after commissioning complete for ICD device,  we can show the UserActiveMode in the UI, please follow-up it, thanks